src/views/Developers.vue

```diff
@@ -1,16 +1,21 @@
 <template>
   <div>
     <SubNav :sections="sections" />
-    <router-view />
+    <router-view>
+      <InputSearch label="Filter developer by name or login name..." />
```

Comment:
nitpick: **login** is the internal keyword they use to refer to the username, so let change it to **username**
```suggestion
      <InputSearch label="Filter developer by name or username..." />
```